.github/workflows/flutter-candidate-update.yaml

```diff
@@ -0,0 +1,68 @@
+name: Flutter Candidate Update
+on:
+  workflow_dispatch: # Allows for manual triggering if needed
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 8 * * *" # Run every day at midnight Pacific Time
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  update-candidate:
+    name: Bump Version
```

Comment:
Can you name this 'Update Flutter Candidate Version' or similar?

Reply:
oops ty!